docs/content/standards/display.mdx

```diff
@@ -44,75 +44,77 @@ The following represents the template the `init` function defines:
 /// mint their Hero. Shows how to initialize the `Publisher` and how
 /// to use it to get the `Display<Hero>` object - a way to describe a
 /// type for the ecosystem.
-module examples::my_hero {
-    use sui::tx_context::{sender};
-    use std::string::{utf8, String};
-
-    // The creator bundle: these two packages often go together.
-    use sui::package;
-    use sui::display;
-
-    /// The Hero - an outstanding collection of digital art.
-    public struct Hero has key, store {
-        id: UID,
-        name: String,
-        image_url: String,
-    }
+module examples::my_hero;
```

Comment:
Would be nice to have all this code in the `examples` folder. Will make a separate ticket for that.